tests/regression/evaluate_rag_retrieval.py

```
@@ -219,17 +382,29 @@ def main() -> int:
     ap = argparse.ArgumentParser(description="Evaluate RAG retrieval and source-grounded answer quality")
     ap.add_argument(
         "--cases",
-        default=str(Path(__file__).with_name("rag_eval_cases.json")),
-        help="path to RAG eval cases JSON",
+        default=str(DEFAULT_CASES_PATH),
+        help="path to a RAG eval cases JSON file or a directory of *.json files",
     )
     ap.add_argument("--top-k", type=int, default=5, help="hybrid_search top_k for retrieval metrics")
     ap.add_argument("--answer-top-k", type=int, default=6, help="build_answer top_k for answer metrics")
     ap.add_argument("--schedule-top-k", type=int, default=5, help="schedule_search top_k for schedule cases")
     ap.add_argument("--out", default="/tmp/rag_eval_report.json", help="output report path")
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/rag_eval_report.json"

(S108)